sources/packages/backend/libs/test-utils/src/factories/institution-location.ts

```diff
@@ -0,0 +1,43 @@
+import { Institution, InstitutionLocation } from "@sims/sims-db";
+import * as faker from "faker";
+
+export function createFakeInstitutionLocation(
+  institution: Institution,
+): InstitutionLocation {
+  const institutionLocation = new InstitutionLocation();
+
+  institutionLocation.name = faker.company.companyName();
+  institutionLocation.institution = institution;
+  institutionLocation.data = {
+    address: {
+      addressLine1: faker.address.streetAddress(),
+      city: "Victoria",
+      provinceState: "BC",
+      country: "canada",
+      selectedCountry: "Canada",
+      postalCode: faker.address.zipCode("A9A9A9"),
+    },
+  };
+  institutionLocation.primaryContact = {
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email(),
+    phone: faker.phone.phoneNumber("##########"),
+  };
+  institutionLocation.institutionCode = faker.random.alpha({
```

Comment:
Not part of the PR, but need to be generated in a controlled way to ensure that it is unique.
@velang01 FYI